parsl/tests/test_error_handling/test_resource_spec.py

```diff
@@ -0,0 +1,30 @@
+import parsl
+from parsl.app.app import python_app
+from parsl.tests.configs.local_threads import config
+from parsl.executors.errors import UnsupportedFeatureError
+from parsl.executors import WorkQueueExecutor
+
+local_config = config
```

Comment:
is this intended to be a local test or a test run on arbitrary configs? (the second, i think)

Reply:
Yes, the second. I am removing this line.